forgerock-openbanking-tpp-core/src/main/java/com/forgerock/openbanking/tpp/core/services/directory/DirectoryService.java

```diff
@@ -58,7 +58,7 @@ public ApplicationIdentity authenticate(JWK jwk) {
         headers.setContentType(MediaType.APPLICATION_JSON);
 
         ParameterizedTypeReference<ApplicationIdentity> ptr = new ParameterizedTypeReference<ApplicationIdentity>() {};
-        HttpEntity<String> request = new HttpEntity<>(jwk.toJSONObject().toJSONString(), headers);
+        HttpEntity<String> request = new HttpEntity<>(jwk.toJSONObject().toString(), headers);
```

Comment:
JSONObjectUtils.toJsonString(jwk.toJSONObject()) would be better.

Reply:
True! and Done!